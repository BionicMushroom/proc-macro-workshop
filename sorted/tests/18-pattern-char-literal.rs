#[sorted::check]
fn main() {
    let x = 'a';

    #[sorted]
    match x {
        'd' => println!("d"),
        'a' => println!("a"),
        _ => println!("other"),
    }
}
