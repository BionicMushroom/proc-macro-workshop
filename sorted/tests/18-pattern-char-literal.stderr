error: 'a' should sort before 'd'
 --> tests/18-pattern-char-literal.rs:8:9
  |
8 |         'a' => println!("a"),
  |         ^^^
